### PR TITLE
OEP-0019: Developer Documentation

### PR DESCRIPTION
**Pull request by grantgoodmanedX**

### CODE REVIEW

File: oeps/oep-0019-developer-documentation.rst [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+maintained. Documents exist on ReadTheDocs (RTD), in repositories on GitHub,
+in Confluence, and in Google Docs.
+
+*************
+Specification
+*************
+
+We should distinguish between documentation that is strictly internal to edX
+operations and documentation that is helpful for any Open edX platform
+develoers. Documentation for the Open edX platform should be maintained in the
```
Comment (OmarIthawi) [inline oeps/oep-0019-developer-documentation.rst]: `develoers` -> `developers`

**Comment by OmarIthawi** [comment]
@grantgoodmanedX Nice an succinct OEP. I like it. One big suggestion though:

 - Please, try to keep as much as of the edX-internal documentation open.

I would go further and ask, please make it explicit to follow an open-by-default documentation policy. You have no idea how community members like me benefit from those docs. For example [those architecture docs](https://openedx.atlassian.net/wiki/spaces/AC/pages/66584838/Architecture+Design+Documents) are very helpful to know what to expect in future changes in platform.

**Comment by OmarIthawi** [comment]
@grantgoodmanedX @nasthagiri @e0d is there any intention to move this OEP forward?

**Comment by nasthagiri** [comment]
@OmarIthawi Thanks for asking.  Yes, @robrap is planning to take this forward over the next few days.

**Comment by robrap** [comment]
This PR was replaced by https://github.com/edx/open-edx-proposals/pull/74, which updates the branch name to match to avoid confusion.

**Pull request closed without merging**